calcom/cal.com
# feat: Add Eastern and Eastern Pacific time zones  #16880

## What does this PR do?

Added Eastern Time - US Canada, Pacific Time , etc
In total 15 time zones are added.

- Fixes #16880
- Fixes CAL-4434



## Mandatory Tasks (DO NOT REMOVE)

- [ YES] I have self-reviewed the code (A decent size PR without self-review might be rejected).
- [N/A ] I have added a Docs issue [here](https://github.com/calcom/docs/issues/new) if this PR makes changes that would require a [documentation change](https://docs.cal.com/docs). If N/A, write N/A here and check the checkbox.
- [ YES] I confirm automated tests are in place that prove my fix is effective or that my feature works.

## How should this be tested?

visit any page which has time zones options in it and you can see time zones like Eastern Time - US Canada, Central TIme, Mountain Time, Indian Standard TIme, etc.

- Are there environment variables that should be set? NO
- What are the minimal test data to have? No specific test data required
- What is expected (happy path) to have (input and output)?  Getting newly added time zones in time zones dropdowns 
- Any other important info that could help to test that PR. NO
